src/test/groovy/jetbrains/buildServer/auth/oauth/OAuthUserTest.groovy

```diff
@@ -20,16 +21,17 @@ class OAuthUserTest extends Specification {
         then:
         user.id == expectedId
         user.name == expectedName
+        user.groups == expectedGroups
         where:
-        location                        || expectedId             | expectedName
-        "classpath:user/github.json"    || 'pwielgolaski'         | 'Piotr Wielgolaski'
-        "classpath:user/bitbucket.json" || 'pwielgolaski'         | 'Piotr Wielgolaski'
-        "classpath:user/google.json"    || 'superemail@gmail.com' | 'Piotr Wielgołaski'
-        "classpath:user/azure.json"     || 'any-guid'             | 'Piotr Wielgołaski'
+        location                        || expectedId             | expectedName        | expectedGroups
+        "classpath:user/github.json"    || 'pwielgolaski'         | 'Piotr Wielgolaski' | ["dev"]
+        "classpath:user/bitbucket.json" || 'pwielgolaski'         | 'Piotr Wielgolaski' | ["dev"]
+        "classpath:user/google.json"    || 'superemail@gmail.com' | 'Piotr Wielgołaski' | ["dev"]
+        "classpath:user/azure.json"     || 'any-guid'             | 'Piotr Wielgołaski' | ["dev"]
```

Comment:
public services does not provide groups, so please dont extend responses for groups, it should be empty
if you have custom oauth that support it, add extra response sample